Bottom: f10ff6f9972da1c57b4cf82c6ba9ea5d15b357c6
Top:    5ec1da356d5808f2fa755ce562b04eaf2f7eb750
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-06-15 13:34:46 +0800

Refresh of tun-introduce-tx-array-mode

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 8e49d97..20bebc7 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1885,7 +1885,7 @@ static int tun_set_iff(struct net *net, struct file *file, struct ifreq *ifr)
 
 		tun = netdev_priv(dev);
 		tun->dev = dev;
-		tun->flags = flags | IFF_TX_ARRAY;
+		tun->flags = flags;
 		tun->txflt.count = 0;
 		tun->vnet_hdr_sz = sizeof(struct virtio_net_hdr);
